src/analysis/calculus/fderiv.lean

```diff
@@ -2867,4 +2867,37 @@ lemma differentiable.restrict_scalars (h : differentiable 𝕜' f) :
   differentiable 𝕜 f :=
 λx, (h x).restrict_scalars 𝕜
 
+lemma has_fderiv_within_at_of_restrict_scalars {s : set E}
```

Comment:
```suggestion
lemma has_fderiv_within_at_of_restrict_scalars
```
There's already an `s` in scope, I think, and making use of that one ensures that the argument order is "standard".  Same for the lemma `differentiable_within_at_iff_restrict_scalars` below.